PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Data/Data/HospitalDbContext.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using PrivateHospitals.Core.Models;
+
+namespace PrivateHospitals.Data.Data;
```

Comment:
Rename the project to Infrastructure. It's not only about Data, but here you will also have Event broker Messaging, Identity concerns, GRPC handlers, etc. It's not only Data access layer, but something bigger

Reply:
Fixed